.github/workflows/docbuild.yml

```diff
@@ -10,6 +10,8 @@ on:
     branches:
       - main
       - 'v*-branch'
+    tags:
+      - v*
```

Comment:
But are we sure we want to run this on the tag? I thought we had some post-tag tasks to do manually before publishing the doc? @greg-fer @b-gent @umapraseeda

Reply:
I don't think we have a better candidate than tag if we want to automate.

Reply:
@carlescufi , the version PRs are added before the final tagging and doc upload. The post tag tasks are done on main to revert the version on 'latest'.

Reply:
There are no post-tag tasks required before publishing the doc, we publish the tag version.